client/app/lib/calculateTextWidth.ts

```diff
@@ -0,0 +1,16 @@
+const canvas = document.createElement("canvas");
+canvas.style.display = "none";
+document.body.appendChild(canvas);
+
+export function calculateTextWidth(text: string, container = document.body) {
+  const ctx = canvas.getContext("2d");
+  if (ctx) {
+    const containerStyle = window.getComputedStyle(container);
+    ctx.font = `${containerStyle.fontSize} ${containerStyle.fontFamily}`;
```

Comment:
I don't really like this, but for some reason the font property returns an empty string

Reply:
Hm, interesting 🤔  At least from devtools it works just fine. Can you check it once more?

![image](https://user-images.githubusercontent.com/12139186/96958603-ad839180-1506-11eb-9f1b-7632a6dc25ce.png)

Reply:
It might be a Firefox thing, take a look: 
![image](https://user-images.githubusercontent.com/44540213/96997531-5088db00-1508-11eb-9f7f-807f999b919f.png)

Reply:
That also raised my brows to the possibility of having browser inconsistencies with the canvas element. Wdyt?

Reply:
It's not about the canvas @rafawendel:
<img width="735" alt="Screen Shot 2020-10-23 at 10 24 27" src="https://user-images.githubusercontent.com/3356951/97009063-04469680-151a-11eb-8ed5-dac71d0fed11.png">
[getComputedStyle docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle)

Reply:
BTW doesn't it work with:
```js
ctx.fontSize = containerStyle.fontSize;
ctx.fontFamily = containerStyle.fontFamily;
```
?

Reply:
Unfortunately not